tests/console/libssh.pm

```diff
@@ -0,0 +1,151 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: libssh4 libssh2-1
+# Summary: libssh version 1 and version 2 test
+#  Libssh4 or libssh2-1 are used by the following softwares respectively
+#  so they are tested with them.
+#    libssh: curl, libvirt
+#    libssh2: qemu-block-ssh, libvirt
+#  Steps:
+#  * Set up a docker container as one end of client/server test
+#  * Run below commands to connect to sshd deamon.
+#      curl -s sftp://
+#      curl -s scp://
+#      qemu-system-x86_64 -drive file=ssh://
+#  * Run below commands to connect to libvirtd deamon.
+#      virsh -c "qemu+libssh://"
+#      virsh -c "qemu+libssh2://"
+#  * It's difficult to clean up libvirt so set test flag to rollback always
+#
+# Maintainer: Tony Yuan <tyuan@suse.com>
+
+package libssh;
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Systemd 'disable_and_stop_service';
+use version_utils;
+use registration 'add_suseconnect_product';
+
+# Build a custom container image with openssl, curl and virsh installed.
+# The container will be used as client of libssh
+# The variable: dev_image_tag can be used to specify a base image different from host OS. If the base images is
+# stored in internel repository i.e registry.suse.de we assume the host OS is in develping stage i.e currently sle15sp3:
+# registry.suse.de/suse/sle-15-sp3/update/cr/totest/images/suse/sle15:15.3
+#
+sub create_image {
+    my $tag = get_var("DEV_IMAGE_TAG");
+    my $kubic;
+    if ($tag =~ /registry\.suse\.de/) {
+        assert_script_run("curl http://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/pki/trust/anchors/SUSE_Trust_Root.crt");
```

Comment:
You can use `utils::ensure_ca_certificates_suse_installed`

Reply:
Will use ensure_ca_certificates_suse_installe.